pallets/starknet/src/lib.rs

```diff
@@ -193,8 +193,31 @@ pub mod pallet {
 		/// * Implement the function.
 		fn store_block(block_number: U256) {
 			// TODO: Use actual values.
+			let parent_block_hash = U256::zero();
+
+			let global_state_root = U256::zero();
 			let sequencer_address = U256::zero();
-			let block = Block::new(Header::new(block_number, sequencer_address));
+			let block_timestamp = 0_u128;
+			let transaction_count = 0_u128;
+			let transaction_commitment = U256::zero();
+			let event_count = 0_u128;
+			let event_commitment = U256::zero();
+			let protocol_version = None;
+			let extra_data = None;
+
+			let block = Block::new(Header::new(
```

Comment:
nit: consider using builder pattern, to avoid accumulating more and more parameters in new method.

https://crates.io/crates/derive_builder